nixpkgs_review/cli/__init__.py

```diff
@@ -223,6 +223,13 @@ def common_flags() -> List[CommonFlag]:
             default=read_github_token(),
             help="Github access token (optional if request limit exceeds)",
         ),
+        CommonFlag(
+            "--build-graph",
```

Comment:
How about this name?

```suggestion
            "--build-program",
```

Reply:
I like the `graph` more, since that is the main feature why we'd use `nom`.  `--build-program` sounds like it will build the derivation with something different (which isn't true. It is just a pipe from `nix` to `nom`)